src/webgl/shaders/filters/blur.frag

```diff
@@ -8,24 +8,58 @@ precision highp float;
 uniform sampler2D tex0;
 varying vec2 vTexCoord;
 uniform vec2 direction;
-uniform vec2 texelSize;
-uniform float steps;
+uniform vec2 canvasSize;
+uniform float radius;
 
-void main(){
-  const float maxIterations = 100.0;
+float random2 (vec2 st) {
```

Comment:
Do we need this function or can it be removed?

Reply:
Oops good catch, I'll take this out!